src/asciidoc/docs/guides-general/capture-user-input.adoc

```diff
@@ -0,0 +1,319 @@
+= Capturing user input
+:page-slug: capture-user-input
+:page-description: How to capture user input using `Koji.ui.capture` controls.
+:includespath: ../_includes
+:imagesdir: path-to-images-directory
+
+Koji provides a set of predefined user-input controls that you can use in your app's frontend to capture information and content from your users.
+
+These controls are found in `Koji.ui.capture` in the @withkoji/core package.
+
+== Using user-input controls
+
+To use a predefined user-input control in your app,
+
+* Install @withkoji/core in the *frontend* terminal tab.
++
+.Learn how...
+[%collapsible]
+====
+include::{includespath}/steps-codeeditor.adoc[tag=cli-front-end]
+include::{includespath}/steps-codeeditor.adoc[tag=terminal-stop-process]
+include::{includespath}/steps-codeeditor.adoc[tag=terminal-install-withkoji-core-package]
+include::{includespath}/steps-codeeditor.adoc[tag=terminal-restart-process]
+====
+
+* Import @withkoji/core in your front-end code.
++
+[source,javascript]
+----
+import Koji from '@withkoji/core';
+----
+
+* Call the appropriate `Koji.ui.capture` control.
++
+[source,javascript]
+----
+myVariable = await Koji.ui.capture.<control-name>( <control-parameters> )
+----
++
+** `.audio` – Prompts the user to select or upload an audio file.
```

Comment:
As discussed, let's try embedding the production app in an iframe on the page. That way, we have a live demo and don't need all the screenshots. Then, we can just include a code sample for one use case as an example, rather than having one for each method in a collapsible section. 

When you embed the app, you can talk about it and how to use it to see the controls. Highlights your cool app more!

Reply:
- Embedded the example app in a new "Example" section with only one set of code snippets (showing the `.image` parts of the code). Added a link so they can fork the project to see the full code.
- Removed the screenshots and code snippets from the list.

FYI: The popup menus are different when viewed within the iframe and when viewed within the code editor or from the browser, even if I use the same production URL (https://withkoji.com/@eltesorokoji/iwmb) in the iframe and in the browser address bar. The iframe includes an additional option to "Take photo" or "Capture audio".